cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -288,7 +288,11 @@ def _validate(self, instance_type, efa_enabled, gdr_support):
 class EfaPlacementGroupValidator(Validator):
     """Validate placement group if EFA is enabled."""
 
-    def _validate(self, efa_enabled, placement_group_enabled, placement_group_config_implicit):
+    def _validate(self, efa_enabled, placement_group):
+        placement_group_enabled = placement_group and (placement_group.enabled or placement_group.id)
```

Comment:
This will evaluate to `true` if the PlacementGroup/enabled is set to `false` but the name/id is specified, right?